lib/Listerine/helpers.ex

```diff
@@ -1,30 +1,10 @@
 defmodule Listerine.Helpers do
   use Coxir
-
-  def add_role_list(message, role_names) do
-    guild = message.channel.guild_id
-    roles = Guild.get_roles(guild)
-
-    if role_names != [] do
-      [head | tail] = role_names
-      add_role_name(head, message, roles, guild)
-      add_role_list(message, tail)
-    end
+  
+  def merge(a, b, c) do
+    merged = Map.merge(a, b)
+    Map.merge(merged, c)
   end
```

Comment:
I don't think abstracting this to a function is necessary as it can be done in one line `Map.merge(a, b) |> Map.merge(c)`